website/content/docs/agent/index.mdx

```diff
@@ -9,33 +9,66 @@ description: >-
+The agent maintains membership information, registers services, runs checks, responds to queries, and more.
+The agent must run on every node that is part of a Consul cluster.
+
+Agents run in either client or server mode. Client nodes are lightweight processes that make up the majority of the cluster.
+They interface with the server nodes for most operations and maintain very little state of their own.
+Clients run on every node where services are running.
+
+In addition to the core agent operations, server nodes participate in the [consensus quorum](/docs/internals/consensus).
+The quorum is based on the Raft protocol, which provides strong consistency and availability in the case of failure.
+Server nodes should run on dedicated instances because they are more resource intensive than client nodes.
+
+## Requirements
+
+You should run one Consul agent per server or host.
+Instances of Consul can run in separate VMs or as separate containers.
+At least one server agent per Consul deployment is required, but three to five server agents are recommended.
+Refer to the following sections for information about host, port, memory, and other requirements:
+
+- [Server Performance](/docs/install/performance)
+- [Required Ports](/docs/install/ports)
+
+The [Datacenter Deploy tutorial](https://learn.hashicorp.com/tutorials/consul/reference-architecture?in=consul/production-deploy#deployment-system-requirements) contains additional information, including licensing configuration, environment variables, and other details.
+
+## Starting the Consul Agent
+
+Start a Consul agent with the `consul` command and `agent` subcommand using the following syntax:
+
+```shell-session
+consul agent <options>
+```
+
+Consul ships with a `-dev` flag that configures the agent to run with several additional settings.
+The `-dev` flag is provided for learning purposes only.
+We strongly advise against using it for production environments.
+
+-> **Getting Started Tutorials**: You can test a local agent by following the
 [Getting Started tutorials](https://learn.hashicorp.com/tutorials/consul/get-started-install?utm_source=consul.io&utm_medium=docs).
 
-The agent command takes a variety of
-[`configuration options`](/docs/agent/options#command-line-options), but most
-have reasonable defaults.
+The only information Consul needs to run is the location of a directory for storing agent state data, specified with the `-data-dir` flag.
+Specifying the data directory and no other options will start a client agent, but the agent will be unable to perform any operations or connect to any other nodes.
+
+The following example starts a Consul agent in dev mode that will store agent state data in the `tmp/consul` directory relative to the current directory:
+
+```shell-session
+consul agent -config-file=client.hcl -dev
+```
+
+In the following example, the agent configuration file would contain the following setting:
```

Comment:
This part confused me. Is the intent to showcase the ability to specify another directory for the data location?

Reply:
Yes, I was trying to express that the agent can be configured in an external file. Is it confusing at this point? Should we just link to the section that talks about config files?